webpack-test/README.md

```diff
@@ -70,3 +73,11 @@ If you are still nervous or don't quite understand, please submit an issue and t
 
 ## Footnotes
 <sup>1</sup> webpack's parser supports the use of ES2015 features like arrow functions, harmony exports, etc. However as a library we follow Node.js' timeline for dropping older versions of node. Because of this we expect your tests on GitHub Actions to pass all the way back to NodeJS v10; Therefore if you would like specific tests that use these features to be ignored if they are not supported, then you should add a `test.filter.js` file. This allows you to import the syntax needed for that test, meanwhile ignoring it on node versions (during CI) that don't support it. webpack has a variety of helpful examples you can refer to if you are just starting out. See the `./helpers` folder to find a list of the versions.
+
+
+## Credits
```

Comment:
Should we also place the license here(in this folder)?

Reply:
Not sure, but we already have licenses on our homepage `README.md`

Reply:
I see. You might need to change the following line to declare what parts are referenced.

https://github.com/web-infra-dev/rspack/blob/cc142094d2e9bbaea26d68d7e213ddf7738488cd/LICENSE#L24

Reply:
> I see. You might need to change the following line to declare what parts are referenced.
> 
> https://github.com/web-infra-dev/rspack/blob/cc142094d2e9bbaea26d68d7e213ddf7738488cd/LICENSE#L24
but `webpack/test` is already part of webpack, I am not sure if it is necessary to claim that.

Reply:
I am not sure what you mean.

My question is: is the current LICENSE description clearly show what parts of the Rspack repo are copied from the Webpack repo?

And my answer is we don't seem to make a clear declaration in the LICENSE for the code this PR migrated.

Reply:
@hardfist

Reply:
My suggestions to deal with the question are: 

- Place the webpack license in the `webpack-test` folder

or

- Make some declarations in https://github.com/web-infra-dev/rspack/blob/main/LICENSE

Reply:
I think maybe we could discuss this in another place, IIRC,  there is plenty of code copy from `webpack` repo, but do not claim in our license file.

Reply:
> My suggestions to deal with the question are:
> 
> * Place the webpack license in the `webpack-test` folder
> 
> or
> 
> * Make some declarations in https://github.com/web-infra-dev/rspack/blob/main/LICENSE

OK

Reply:
I'm ok with this. You might need the final review of @hardfist.